backstop/tests/table.html

```diff
@@ -1775,104 +1781,134 @@ <h2>Additional options</h2>
         style="max-height: 407px"
         id="demo-extras-scrollable-table"
       >
-        <div class="iui-table-header">
-          <div class="iui-row">
-            <div class="iui-cell iui-actionable iui-sorted">
-              Product
-              <button
-                class="iui-button iui-borderless iui-filter-button"
-                aria-label="Apply filter"
-                type="button"
+        <div class="iui-table-header-wrapper">
+          <div class="iui-table-header">
+            <div class="iui-row">
+              <div
```

Comment:
I didn't want for first tables to have too much functionality so I added some stuff to these ones.

Reply:
Added